oslabs-beta/vista
# QueryCard Component that renders the type name on each card

## Overview

**Issue Type**


**Description**
QueryCard component and FieldsBtn component are two components that work together that pull data from DisplayData and render the Type name from each Type that QueryCard generates and displays our FieldsBtn component on each card. You should see a card component with a Type name that is clickable. The scope of this case wasn't to generate any onClick logic, but mainly to render the name of the Type on each card.

**Ticket Item**
Trello Card click [here](https://trello.com/c/crahExbk/18-grab-the-data-sent-from-the-backend-and-render-a-container-for-each-type-example-one-for-books-and-another-for-authors)


**Steps to Validate Feature **

1. Insert a valid query in the input box and hit enter
2. You should see individaul cards rendered with the correct name for each Type. For example, if your Schema has 10 different Types. You should see 10 cards rendered and each one with the label of the Type name.


**Previous behavior**
Before if you clicked enter after hitting a valid query, nothing was returned or rendered.

**Expected behavior**
Inserting a query will generate cards for each type
